hphp/runtime/ext/pdo_mysql.cpp

```diff
@@ -1284,22 +1284,29 @@ bool PDOMySqlStatement::nextRowset() {
     return false;
   }
 
-  my_ulonglong row_count;
+  my_ulonglong affected_count;
   if (!m_conn->buffered()) {
     m_result = mysql_use_result(m_server);
-    row_count = 0;
+    affected_count = 0;
   } else {
     m_result = mysql_store_result(m_server);
-    if ((my_ulonglong)-1 == (row_count = mysql_affected_rows(m_server))) {
+    if ((my_ulonglong)-1 == (affected_count = mysql_affected_rows(m_server))) {
       handleError(__FILE__, __LINE__);
       return false;
     }
   }
-
+  row_count = affected_count;
```

Comment:
I believe it's here: https://github.com/facebook/hhvm/blob/master/hphp/runtime/ext/pdo_driver.h#L649 :)